checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -128,11 +124,19 @@ ValueAnnotatedTypeFactory getValueAnnotatedTypeFactory() {
      * Provides a way to query the Search Index Checker, which helps the Index Checker type the
      * results of calling the JDK's binary search methods correctly.
      */
-    private SearchIndexAnnotatedTypeFactory getSearchIndexAnnotatedTypeFactory() {
+    SearchIndexAnnotatedTypeFactory getSearchIndexAnnotatedTypeFactory() {
```

Comment:
I don't see anywhere else in this PR that needs a `SearchIndexAnnotatedTypeFactory`; maybe this is left over from before you factored out the IndexOf checker?